application/wrapper/storage.go

```diff
@@ -20,8 +20,14 @@ func NewStorage(storageInter dynamics.StorageGetter) *Storage {
 }
 
 // GetMaxBytes returns MaxBytes
-func (s *Storage) GetMaxBytes() uint32 {
-	return s.storage.GetMaxBytes()
+func (s *Storage) GetMaxBytes() (uint32, error) {
+	if s == nil {
+		return 0, errorz.ErrInvalid{}.New("storage.GetAtomicSwapFee; struct not initialized")
+	}
+	if s.storage == nil {
+		return 0, errorz.ErrInvalid{}.New("storage.GetAtomicSwapFee; storage not initialized")
+	}
```

Comment:
Agree with above but this does not ensure database integrity. I think this is a bigger problem we must address in some other way. Lets do an external discussion. As for the specific error types, I think something other than invalid would be wise here given this should unwind the main and invalid is often caught up above in the gossip bus and otherwise. For now lets change the error types and merge this for coverage knowing we must discuss clean exit.

Reply:
Works for me, we can take offline.